telosnetwork/open-block-explorer
# #540 | Click animation

# Fixes #540

## Description
This PR updates the UI for inputs in the account section, specifically adding color and tooltips to the places where balance is displayed and clicking the balance auto fills the maximum amount. 

## Test scenarios

- go to https://deploy-preview-563--obe-staging.netlify.app/
- sign into an account, e.g. cleos -> `eztestnet123`
- go to the account page, e.g. https://deploy-preview-563--obe-staging.netlify.app/account/eztestnet123
- click `Send`
    - you should see a tooltip info icon next to the balance total above the input
- hover over the balance total
    - the total should turn blue and a tooltip should appear
- go back to account page
- click `Staking (REX)`
- repeat those steps for each input in the following tabs: Stake, Unstake, Stake from CPU/NET, Savings

## Checklist:
